assets/js/_js/projects/read-more-less.js

```diff
@@ -9,18 +9,10 @@
   $(function () {
     let readMoreLess = $(".read-more-less");
     readMoreLess.click(function () {
-      var titleSubtitleUbication = $(this).parent().children();
-      for (var i = 0; i < 3; i++) {
-        document.getElementById("principalPopUp").appendChild(titleSubtitleUbication[i].cloneNode(true));
-      }
-      var description = $(this).parent().parent().parent().find(".markdown-style").children();
-      var descriptionContainer = document.getElementById("descPopUp");
-      while (descriptionContainer.firstChild) {
-        descriptionContainer.removeChild(descriptionContainer.firstChild);
-      }
-      for (var i = 0; i < description.length; i++) {
-        descriptionContainer.appendChild(description[i].cloneNode(true));
-      }
+      var titleSubtitleUbication = $(this).siblings(":header").clone();
+      $("#principalPopUp").empty().append(titleSubtitleUbication);
+      var description = $(this).siblings(".markdown-style").children().clone();
```

Comment:
The name of the class `markdown-style` is not descriptive. Also you're using the siblings for the title and here the children. It would be nice to use a similar approach in both.